.circleci/config.yml

```diff
@@ -140,17 +133,7 @@ jobs:
       - run:
           name: Deploy
           command: |
-            echo "[distutils]" > ~/.pypirc
-            echo "index-servers =" >> ~/.pypirc
-            echo "    pypi " >> ~/.pypirc
-            echo >> ~/.pypirc
-            echo "[pypi]" >> ~/.pypirc
-            echo "repository:https://upload.pypi.org/legacy/" >> ~/.pypirc
-            echo "username:$PYPI_USERNAME" >> ~/.pypirc
-            echo "password:$PYPI_PASSWORD" >> ~/.pypirc
-            echo >> ~/.pypirc
-            python setup.py register -r pypi
-            python setup.py sdist upload -r pypi
+            echo "Not publishing package!"
```

Comment:
??

Reply:
this is the autogenerated output... that's odd.

Reply:
The `pypi` key changed. 

you need to add:

```
"pypi": {
  "repository": "pypi"
}
```